src/components/Modal/Modal.js

```diff
@@ -102,8 +104,52 @@ const Modal = ({ showModal, layers, lastClickData, newsModal, onCloseModal, onGe
                             Litologia do Estado do Rio de Janeiro na escala 1:50000
                             <img src={require('../../assets/img/news/geologia.png')} alt="" className="news-modal-list--image"/>
                         </dd>
-                    </dl>
-
+                    </dl> */}
+
+                    <ol className="news-modal-list">
```

Comment:
I changed because of the need to need more elements to align in a better way.
The code would no longer be semantic to use this tag per account in the need to tinker in the positioning of the elements on the screen and etc ...